crates/turborepo-scm/src/git.rs

```diff
@@ -350,15 +360,29 @@ impl GitRepo {
         files: &mut HashSet<AnchoredSystemPathBuf>,
         turbo_root: &AbsoluteSystemPath,
         stdout: Vec<u8>,
-    ) {
-        let stdout = String::from_utf8(stdout).unwrap();
+    ) -> Result<(), Error> {
+        let stdout = String::from_utf8_lossy(&stdout);
         for line in stdout.lines() {
-            let path = RelativeUnixPath::new(line).unwrap();
-            let anchored_to_turbo_root_file_path = self
-                .reanchor_path_from_git_root_to_turbo_root(turbo_root, path)
-                .unwrap();
-            files.insert(anchored_to_turbo_root_file_path);
+            match RelativeUnixPath::new(line) {
```

Comment:
We do not need to handle the error case for `RelativeUnixPath::new` failing. We can just `unwrap()` it as it will only fail if given a path that starts with `/` and `git` will not output absolute paths for the command we're parsing.